core/controllers/editor.py

```diff
@@ -365,67 +365,52 @@ def put(self, exploration_id):
 class ExplorationStatusHandler(EditorHandler):
     """Handles publishing or publicizing of an exploration."""
 
-    def _publish_exploration(self, exploration_id, make_public):
-        """Publish/Unpublish an exploration based on make_public boolean.
+    def _publish_exploration(self, exploration_id):
+        """Publish an exploration based.
```

Comment:
Yes. (Also, just FYI: https://english.stackexchange.com/questions/66487/meaning-of-scan-in-that-doesnt-scan-in-english)